hotload.py

```diff
@@ -135,6 +150,13 @@ class ClearTerminal(Runnable):
     def run(self):
         os.system("cls" if os.name == "nt" else "clear")
 
+class ReloadModules(Runnable):
```

Comment:
perhaps not really a `Runnable` when `run` requires an additional argument?

Reply:
I'm open to allowing `run` accept (*args, **kwargs). In that case, we could allways provide a list of modules.

```python
step.run(changed_modules=changed_modules)
```

---

Though if the list of modules is passed to ReloadModules on __init__, perhaps it doesn't need any special `run()` treatment?

Reply:
Edit - I see that you're only reloading modules that have changed. In that case, we'd need to pass the list of changed modules.